ovos_utils/sound/__init__.py

```diff
@@ -33,74 +34,50 @@ def _get_pulse_environment(config):
         return os.environ
 
 
+def _play_default_sound_locally(sound_name):
+    audio_file = resolve_resource_file(
+        read_mycroft_config().get('sounds', {}).get(sound_name))
+    if not audio_file:
+        LOG.warning(f"Could not find '{sound_name}' audio file!")
+        return
+    process = play_audio(audio_file)
+    if not process:
+        LOG.warning(f"Unable to play '{sound_name}' audio file!")
+    return process
+
+
+@deprecated("please emit mycroft.audio.play_sound instead", "0.1.0")
```

Comment:
Can we include a convenience method somewhere? This refactor adds a fair amount of complexity to know that a message should be forwarded for routing compat. and to find out what data structure to pass and where the default sound is in configuration

Reply:
acknowledge is already exposed in the skill base class

the goal is to move all playback to ovos-audio so nothign should really be using these methods directly, the original PR was adding a optional bus object but i'd rather have devs refactor the whole thing to emit the bus message than refactoring to pass bus to these methods... 

there's a bunch of companion PRs for context

Reply:
If there are equivalent methods in the skill class then I think those should be referenced in the deprecation warning since that would directly cover most existing usage and provide an example implementation for non-skills

Reply:
I guess those aren't really direct replacements and in any case the deprecation notice gives devs something to search for in docs/code